generators/client/templates/angularjs/src/main/webapp/app/blocks/config/_translation.config.js

```diff
@@ -23,12 +23,12 @@
         .module('<%=angularAppName%>')
         .config(translationConfig);
 
-    translationConfig.$inject = ['$translateProvider', 'tmhDynamicLocaleProvider'];
+    translationConfig.$inject = ['$translateProvider', 'tmhDynamicLocaleProvider', 'BUILD_TIMESTAMP'];
 
-    function translationConfig($translateProvider, tmhDynamicLocaleProvider) {
+    function translationConfig($translateProvider, tmhDynamicLocaleProvider, BUILD_TIMESTAMP) {
         // Initialize angular-translate
         $translateProvider.useLoader('$translatePartialLoader', {
-            urlTemplate: 'i18n/{lang}/{part}.json'
+            urlTemplate: 'i18n/{lang}/{part}.json' + (BUILD_TIMESTAMP ? '?build=' + BUILD_TIMESTAMP : '')
```

Comment:
To hit the cache and avoiding reload the same file. The translation files don't change for the same deployment (unlike db data for example)..